linux/free: clarify which units are used (#23061)
tldr-pages/tldr: pages/linux/free.md

> Display amount of free and used memory in the system.
> More information: <https://manned.org/free>.

- Display system memory in KiB:

`free`

> - Display memory in Bytes/KiB/MiB/GiB:

Reviewer: ```suggestion
- Display memory in bytes/KiB/MiB/GiB:
```
I believe this would be the correct capitalization.

`free -{{b|k|m|g}}`

> - Display memory alongside the most appropriate human-readable unit (KiB/MiB/GiB etc):

Reviewer: ```suggestion
- Display memory in the most appropriate human-readable units (KiB/MiB/GiB etc):
```
The units are part of the memory display, thus it's not "alongside"

`free {{[-h|--human]}}`

> - Display memory alongside the most appropriate human-readable unit that is a power of 1000 (kB/MB/GB etc):

Reviewer: ```suggestion
- Display memory alongside the most appropriate human-readable unit that is a power of 1000 (KB/MB/GB etc):
```

Reviewer: Nvm, I was mistaking it with kibibyte.

> - Display memory alongside the most appropriate human-readable unit that is a power of 1000 (kB/MB/GB etc):

Reviewer: ```suggestion
- Display memory in the most appropriate human-readable unit that is a power of 1000 (kB/MB/GB etc):
```
